src/cmds/openapi.js

```diff
@@ -98,9 +98,25 @@ exports.run = async function (opts) {
       }
     }
 
+    let bundledSpec;
+    if (spec) {
+      const oas = new OASNormalize(spec, { enablePaths: true });
+      await oas.validate().catch(err => {
+        return Promise.reject(err);
+      });
+      await oas
+        .bundle()
+        .then(res => {
+          bundledSpec = JSON.stringify(res);
+        })
+        .catch(err => {
+          return Promise.reject(err);
+        });
+    }
+
     const options = {
       formData: {
-        spec: fs.createReadStream(path.resolve(process.cwd(), specPath)),
+        spec: bundledSpec || fs.createReadStream(path.resolve(process.cwd(), specPath)),
```

Comment:
I forget what `specPath` is but when would that be present and `spec` (and `bundledSpec`) isn't?

Reply:
Thank you for catching this! I had meant to make the switch from spec to specPath before pushing the initial changes. We should be validating and bundling with specPath instead of spec.